pinot-core/src/main/java/org/apache/pinot/core/transport/ServerChannels.java

```diff
@@ -147,6 +148,9 @@ synchronized void sendRequest(InstanceRequest instanceRequest)
       _channel.writeAndFlush(Unpooled.wrappedBuffer(requestBytes), _channel.voidPromise());
       _brokerMetrics.addMeteredGlobalValue(BrokerMeter.NETTY_CONNECTION_REQUESTS_SENT, 1);
       _brokerMetrics.addMeteredGlobalValue(BrokerMeter.NETTY_CONNECTION_BYTES_SENT, requestBytes.length);
+      String tableName = instanceRequest.getQuery().getQuerySource().getTableName();
```

Comment:
Sorry I misled you here. Not sure why I was thinking this was a server thing.

You can get the request size by taking the query length in BaseRequestHandler, but the query response is another thing. Since we respond with BrokerResponse object, and it is converted to whatever kind of response (JSON in open source), it is trickier. 

Let us discuss

Reply:
By request length, do we mean the query length here?

If it is the wire object's size then InstanceRequest is the correct thing and it is already emitted.

Reply:
@siddharthteotia By "it's already emitted", do you mean NETTY_CONNECTION_REQUESTS_SENT? I think it's a server level metric, and we want to add a table level metric.